tests/Resources/ti.ui.webview.test.js

```diff
@@ -728,4 +729,138 @@ describe('Titanium.UI.WebView', function () {
 		});
 		should(webView).have.a.property('progress').which.is.a.Number(); // should default to 0 until we start loading the page.
 	});
+
+	it.ios('#findString', function (finish) {
+		if (OS_VERSION_MAJOR < 14) {
+			return finish();
+		}
+		win = Ti.UI.createWindow();
+		const webView = Ti.UI.createWebView({
+			url: 'https://www.google.com'
+		});
+
+		webView.addEventListener('load', function () {
+			webView.findString('google', function (e) {
+				if (e.success) {
+					finish();
+				} else {
+					finish(e.error);
+				}
+			});
+		});
+
+		win.add(webView);
+		win.open();
+	});
+
+	it.ios('#createPDF', function (finish) {
+		if (OS_VERSION_MAJOR < 14) {
+			return finish();
+		}
+		win = Ti.UI.createWindow();
+		const webView = Ti.UI.createWebView({
+			url: 'https://www.google.com'
+		});
+
+		webView.addEventListener('load', function () {
+			webView.createPDF(function (e) {
+				try {
+					should(e.success).be.true();
+					should(e.data).be.an.Object();
+					should(e.data).have.a.property('apiName').which.eql('Ti.Blob');
+				} catch (err) {
+					return finish(err);
+				}
+				finish();
+			});
+		});
+
+		win.add(webView);
+		win.open();
+	});
+
+	it.ios('#createWebArchive', function (finish) {
+		if (OS_VERSION_MAJOR < 14) {
+			return finish();
+		}
+		win = Ti.UI.createWindow();
+		const webView = Ti.UI.createWebView({
+			url: 'https://www.google.com'
+		});
+
+		webView.addEventListener('load', function () {
+			webView.createWebArchive(function (e) {
+				try {
+					should(e.success).be.true();
+					should(e.data).be.an.Object();
+					should(e.data).have.a.property('apiName').which.eql('Ti.Blob');
+				} catch (err) {
+					return finish(err);
+				}
+				finish();
+			});
+		});
+
+		win.add(webView);
+		win.open();
+	});
+
+	describe.ios('#findString()', function () {
+		it('is a Function', () => {
+			if (OS_VERSION_MAJOR < 14) {
+				return finish();
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.webview.test.js;
  Line: 811;
-->
- :no_entry_sign: tests/Resources/ti.ui.webview.test.js line 811 – 'finish' is not defined. (no-undef)